rustio/src/client.rs

```diff
@@ -82,15 +82,11 @@ impl Client {
         Self::send_get_request(url).unwrap().json().unwrap()
     }
 
-    pub fn get_station_by_id(&self, id: i32) -> Result<Station,&str> {
+    pub fn get_station_by_id(&self, id: i32) -> Option<Station> {
```

Comment:
Why the change from Result to Option? Many errors can happen here (I/O error, network error, wrong ID,...). So it definitely make sense to use Result here.

But I have to admit, correct error handling is here still missing (see readme.md).